src/System.Private.CoreLib/shared/System/Runtime/Loader/AssemblyLoadContext.cs

```diff
@@ -582,18 +582,29 @@ public void Dispose()
             return context.ResolveSatelliteAssembly(assemblyName);
         }
 
-        private Assembly? GetFirstResolvedAssembly(AssemblyName assemblyName)
+        private Assembly? GetFirstResolvedAssemblyFromResolvingEvent(AssemblyName assemblyName)
```

Comment:
@CoffeeFlux The AssemblyContext methods are about to receive a lot of CoreCLR specific tracing. Do you still see value in having them in the shared CoreLib partition?

Reply:
Let's chat at Prague; I haven't kept up to date with CoreCLR plans for loader logging, but it's something I very much want to do in Mono as well.

Reply:
I'm planning to meet with @CoffeeFlux and @lambdageek - this will be one of the topics 😄